app/handlers/config.py

```diff
@@ -19,6 +19,9 @@
 See here for usage examples:
 https://github.com/google/personfinder/wiki/DeveloperFaq
 """
+import sys
+sys.path.append('../app/handlers/')
```

Comment:
Can you avoid appending to sys.path like here, and instead update `import` lines by adding subdirectories?